src_py/__init__.py

```diff
@@ -103,6 +103,11 @@ def warn(self):
         import pygame.cdrom
     except (ImportError, IOError):
         cdrom = MissingModule("cdrom", urgent=1)
+else:
+    if os.environ.get('SDL_VIDEODRIVER') == 'windib':
+        # windib isn't in SDL2, we don't want it to fail.
+        environ['SDL_VIDEODRIVER'] = 'windows'
+
```

Comment:
Should we do this here, or in `display.init()` ? I suspect there may be people who do:

```
import os
import pygame
os.environ['SDL_VIDEODRIVER'] = 'windb'

pygame.display.init()  # or regular pygame.init()
```

Which I think will work with SDL, but will not be caught by this.

Reply:
Good point.

Reply:
Updated it to do the business in `pygame.display.init()` instead.